### ISSUE TITLE
improve installer

### ISSUE THREAD

**Issue by raknjak** [raknjak/CIMembership#4]
any ideas?

**Comment by jodylecompte** [comment]
Here are two things I noticed: 
1) The checks after the initial step 1 compatability checks need to be broken apart and given specific error messages. I made a lucky guess that my issue was file permissions, but I would have had to check several things if that hadn't worked.

2) When installing with a database prefix, I get an error that the table with no prefix doesn't exist, but then if I rename the table then I get the error on validate screen.

For example:
DB Prefix of ci_
Login Page: ci_session is not a table
-Rename table ci_ci_sessions to ci_sessions-
Login Page Works, Post Submit
Error "ci_ci_sessions" is not a table.

This is not an installer issue but inconsistent namings in the models. It seems solid to me otherwise.

**Comment by raknjak** [comment]
I agree that validation needs to improve. Ah, that prefix bug I never solved :|
Thanks for your input.

**Comment by jodylecompte** [comment]
It should be relatively easy to patch up both issues. I'll take a look this weekend. :)
